backend/sublet/models.py

```diff
@@ -42,6 +42,7 @@ class Sublet(models.Model):
     baths = models.IntegerField(null=True, blank=True)
     description = models.TextField(null=True, blank=True)
     external_link = models.URLField(max_length=255)
+    # TODO: change to just one price field and migrateeeee
```

Comment:
Why do we want to range price field? I recall wanting 2 to give a range since the sublet price can be negotiable

Reply:
I think vincent/ashley/I discussed with the other teams and the consensus was the prices were negotiable anyway, so min/max doesn't really serve a purpose. Def open to reconsideration for either way though.